.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+# Build Zotero plugin.
+
+name: Build Zotero OCR
+
+on:
+  push:
+    # branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  # schedule:
+  #  - cron: '21 12 * * 0'
+
+jobs:
+  build-plugin:
+    name: Build Zotero OCR plugin
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup git
+        run: git config --global user.email "ci@example.com" && git config --global user.name "GitHub CI"
+
+      - name: Run build
+        run: ./release.sh $(git describe --tags || echo ci-test) && cp -a build/*.xpi .
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: Zotero OCR plugin (zipped)
+          path: "*.xpi"
```

Comment:
If we could do this without zipping the xpi file, that would be easier to handle.

Reply:
I agree, but that's how GitHub handles artifacts. "There is currently no way to download artifacts in a format other than a Zip or to download individual artifact contents" (see https://github.com/actions/upload-artifact#zip-archives).

Reply:
We could upload the files which make up a xpi file. Then upload-artifacts would create a zip file, but that would not simplify the handling for users: they would have to rename the downloaded zip file to get a xpi file.

Reply:
Maybe I am wrong, but can't these action also handle an individual file upload?  https://github.com/actions/upload-artifact?tab=readme-ov-file#upload-an-individual-file We just need to upload the xpi file which is an individual file.

Reply:
De facto it is an individual file upload, because "*.xpi" is only a single file.